package.json

```diff
@@ -18,54 +19,22 @@
   "types": "lib/peg.d.ts",
   "bin": "bin/peggy",
   "scripts": {
-    "pretest": "npm run parser",
+    "build": "pnpm run -r --if-present --filter ./pkg build",
```

Comment:
Please do not require downstream users to install a secondary package manager to rebuild.  This was a problem with futagoza: many people, myself included, legally cannot install this in many contexts, meaning our ability to patch then test in context is severely impeded.

This monorepo strategy is going to cause damage and does not appear to deliver actual value.

Can we please just do it the normal way 😥  It's so much simpler and doesn't require third party tools that exclude other developers